tests/e2e/utils/workspace/TestWorkspaceUtil.ts

```diff
@@ -123,97 +128,46 @@ export class TestWorkspaceUtil implements ITestWorkspaceUtil {
-        if ( keycloakUrl.substr(7, 4).includes('che')) {
-            const keycloakAuthSuffix = '/auth/realms/che/protocol/openid-connect/token';
-            keycloakUrl = keycloakUrl.replace('che', 'keycloak') + keycloakAuthSuffix;
-            params = {
-                client_id: 'che-public',
-                username: TestConstants.TS_SELENIUM_USERNAME,
-                password: TestConstants.TS_SELENIUM_PASSWORD,
-                grant_type: 'password'
-            };
-        } else {
-            const keycloakAuthSuffix = '/auth/realms/codeready/protocol/openid-connect/token';
-            keycloakUrl = keycloakUrl.replace('codeready', 'keycloak') + keycloakAuthSuffix;
-            params = {
-                client_id: 'codeready-public',
-                username: TestConstants.TS_SELENIUM_USERNAME,
-                password: TestConstants.TS_SELENIUM_PASSWORD,
-                grant_type: 'password'
-            };
-        }
-
+        const workspaceIdUrl: string = `${TestWorkspaceUtil.WORKSPACE_API_URL}/${id}`;
         try {
-            const responseToObtainBearerToken = await axios.post(keycloakUrl, querystring.stringify(params));
-            return responseToObtainBearerToken.data.access_token;
+            const deleteWorkspaceResponse = await this.processRequestHandler.delete(workspaceIdUrl);
+            // response code 204: "No Content" expected
+            if (deleteWorkspaceResponse.status !== 204) {
+                throw new Error(`Can not remove workspace. Code: ${deleteWorkspaceResponse.status} Data: ${deleteWorkspaceResponse.data}`);
+            }
         } catch (err) {
-            console.log(`Can not get bearer token. URL used: ${keycloakUrl}`);
+            console.log(`Removing of workspace failed.`);
             throw err;
         }
-
     }
 
     public async stopWorkspaceById(id: string) {
-        const stopWorkspaceApiUrl: string = `${this.workspaceApiUrl}/${id}/runtime`;
-        try {
-            const stopWorkspaceResponse = await this.processRequest(RequestType.DELETE, stopWorkspaceApiUrl);
+        const stopWorkspaceApiUrl: string = `${TestWorkspaceUtil.WORKSPACE_API_URL}/${id}`;
 
+        try {
+            const stopWorkspaceResponse = await this.processRequestHandler.delete(`${stopWorkspaceApiUrl}/runtime`);
             // response code 204: "No Content" expected
             if (stopWorkspaceResponse.status !== 204) {
                 throw new Error(`Can not stop workspace. Code: ${stopWorkspaceResponse.status} Data: ${stopWorkspaceResponse.data}`);
             }
+
+            for (let i = 0; i < TestConstants.TS_SELENIUM_PLUGIN_PRECENCE_ATTEMPTS; i++) {
+                const wsStatus = await this.processRequestHandler.get(stopWorkspaceApiUrl);
+                if (wsStatus.data.status === 'STOPPED') {
+                    break;
+                }
+                await this.driverHelper.wait(TestConstants.TS_SELENIUM_PLUGIN_PRECENCE_POLLING);
```

Comment:
Typo in the variable, correct is "PRESENCE". :)

Reply:
Fixed in all occurrences.